source/non_matching/coupling.cc

```diff
@@ -959,8 +959,9 @@ namespace NonMatching
           }
       }
   }
-
-#include "coupling.inst"
+#ifndef DOXYGEN
+#  include "coupling.inst"
+#endif
```

Comment:
How is that not a problem in all `.cc` files where we `#include` an `.inst` file?